packages/tests/tests/store/utils.ts

```diff
@@ -114,14 +114,13 @@ export async function startAndConnectLightNode(
   await waitForRemotePeer(waku, [Protocols.Store]);
 
   const wakuConnections = waku.libp2p.getConnections();
-  const nwakuPeers = await instance.peers();
 
-  if (wakuConnections.length < 1 || nwakuPeers.length < 1) {
-    throw new Error(
-      `Expected at least 1 peer in each node. Got waku connections: ${wakuConnections.length} and nwaku: ${nwakuPeers.length}`
-    );
+  if (wakuConnections.length < 1) {
+    throw new Error(`Expected at least 1 connection for js-waku.`);
   }
 
+  await instance.waitForLog(waku.libp2p.peerId.toString(), 100);
```

Comment:
REST endpoint does not recognize `js-waku` as a peer so that is the way we can verify if `nwaku` has connection, not 100% reliably tho but good enough for us

fyi @fbarbu15 @waku-org/js-waku-developers @NagyZoltanPeter